message/model.json

```diff
@@ -35,126 +41,155 @@
             "envelope": {
               "name": "envelope",
               "type": "string",
-              "description": "Message envelope format identifier. This attribute MUST be the same as the 'envelope' attribute of the definition group, but is not automatically inherited because definitions may be cross-referenced across groups",
+              "description": "Message envelope format identifier. This attribute MUST be the same as the 'envelope' attribute of the definition group, but is not automatically inherited because definitions may be cross-referenced across groups. Predefined envelope formats are 'None' and 'CloudEvents/1.0'",
               "ifvalues": {
                 "None": {},
                 "CloudEvents/1.0": {
                   "siblingattributes": {
                     "envelopemetadata": {
                       "name": "envelopemetadata",
                       "type": "object",
-                      "description": "CloudEvents metadata constraints",
+                      "description": "CloudEvents attribute declarations, adding constraints to the CloudEvents envelope not yet covered by the CloudEvents spec https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md",
                       "attributes": {
                         "specversion": {
                           "name": "specversion",
-                          "description": "CloudEvents spec version",
+                          "description": "CloudEvents specversion https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#specversion",
                           "type": "object",
                           "attributes": {
                             "description": {
                               "name": "description",
-                              "type": "string"
+                              "type": "string",
+                              "required": false
```

Comment:
I think it would be better to remove the "required:false" lines to reduce the noise and to not imply that w/o it the default is "required:true".

Reply:
I'm removing them.